lib/cdo/script_constants.rb

```diff
@@ -51,14 +56,43 @@ def ScriptConstants.hoc?(name)
         MINECRAFT_NAME,
         INFINITY_NAME,
         ARTIST_NAME
```

Comment:
I don't actually know what that is, I just tried to keep the behavior and comments here the same.